modules/api-svc/src/index.js

```diff
@@ -210,7 +210,8 @@ class Server extends EventEmitter {
             await this.wso2.auth.start();
         }
 
-        const updateInboundServer = !_.isEqual(this.conf.inbound, newConf.inbound);
+        const updateInboundServer = !_.isEqual(this.conf.inbound, newConf.inbound)
+        || !_.isEqual(this.conf.outbound, newConf.outbound);
```

Comment:
```suggestion
            || !_.isEqual(this.conf.outbound, newConf.outbound);
```

Reply:
Thanks, Not committing this here and I handled this manually to individual extra CI job runs.